mono/mini/mini-arm64.h

```diff
@@ -27,9 +31,15 @@
 
 #define MONO_CONTEXT_SET_LLVM_EXC_REG(ctx, exc) do { (ctx)->regs [0] = (gsize)exc; } while (0)
 
+/*
+* __builtin_frame_address () is broken on some older gcc versions in the presence of
+* frame pointer elimination, see bug #82095.
+*/
 #define MONO_INIT_CONTEXT_FROM_FUNC(ctx,func) do {	\
-		MONO_CONTEXT_SET_BP ((ctx), __builtin_frame_address (0));	\
-		MONO_CONTEXT_SET_SP ((ctx), __builtin_frame_address (0));	\
+        int tmp; \
```

Comment:
I would not change the non-Windows path here. On windows perhaps you can use `_AddressOfReturnAddress`?